### DISCUSSION TITLE
Remove releases disambiguation

### DISCUSSION THREAD

**Discussion by neilanthonystuckey** [RandomNinjaAtk/docker-lidarr-extended#175] [Q&A] [answered]
Hello thanks for your time.  I would like to remove the artist disambiguation on the artist folder names. It's causing some issues with my other apps.  Is this the only section outside of Lidarr's naming i need to change this ? Ichanged Lidarr's naming but its still creating the artist folders with disambiguation in the name

/lidarr/config/extended/scripts/Audio.sh

line 1203 - 1217

# Get Release Titles & Disambiguation
		if [ -f /temp-release-list ]; then
			rm /temp-release-list 
		fi
		for releaseId in $(echo "$lidarrAlbumReleaseIds"); do
			releaseTitle=$(echo "$lidarrAlbumData" | jq -r ".releases[] | select(.id==$releaseId) | .title")
			releaseDisambiguation=$(echo "$lidarrAlbumData" | jq -r ".releases[] | select(.id==$releaseId) | .disambiguation")
			if [ -z "$releaseDisambiguation" ]; then
				releaseDisambiguation=""
			else
				releaseDisambiguation=" ($releaseDisambiguation)" 
			fi
			echo "${releaseTitle}${releaseDisambiguation}" >> /temp-release-list 
			echo "$lidarrAlbumTitle" >> /temp-release-list 
		done

Do i just delete the Get Release Titles & Disambiguation section or would i have to modify the code above to just do the album title only ? 

If i have to modify the code would this work ?

# Get Release Titles & Disambiguation
		if [ -f /temp-release-list ]; then
			rm /temp-release-list 
		fi
		for releaseId in $(echo "$lidarrAlbumReleaseIds"); do
			releaseTitle=$(echo "$lidarrAlbumData" | jq -r ".releases[] | select(.id==$releaseId) | .title")
			fi
			echo "${releaseTitle}" >> /temp-release-list 
			echo "$lidarrAlbumTitle" >> /temp-release-list 
		done

**Comment by RandomNinjaAtk** [comment, marked as answer]
Lidarr handles final destination of media. Making changes to the script won't make a difference because files are managed by Lidarr. You should update/review your Lidarr Media Management settings.
